### PR TITLE
Ports "Converts some swarmer messages to balloon alerts" from AetherStation

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Ports https://github.com/AetherStation/AetherStation13/pull/222 from AetherStation.

Converts some swarmer interaction(integrate and fabricate messages) to balloon alerts. 

# Spriting
Not needed

# Wiki Documentation

Not needed

# Changelog

:cl:  SuperSlayer
tweak: Converted some swarmer messages to balloon alerts
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
Which ones? Please at least _try_ to document your changes.

**Comment by SuperSlayer0** [comment]
> Which ones? Please at least _try_ to document your changes.

:nerd:

**Comment by ghost** [comment]
> Which ones? Please at least _try_ to document your changes.

🤓

File: code/modules/swarmers/swarmer.dm [hunk 1]
```diff
@@ -152,13 +152,15 @@
...
+		balloon_alert(src, "not a suitable location")
 		return
 	if(resources < fabrication_cost)
-		to_chat(src, span_warning("You do not have the necessary resources to fabricate this object."))
+		balloon_alert(src, "not enough resources")
 		return
 	resources -= fabrication_cost
-	return new fabrication_object(drop_location())
+	var/atom/fabricated_object = new fabrication_object(drop_location())
+	fabricated_object.balloon_alert(src, "sucessfully fabricated")
```
Comment (Bibby0110) [inline code/modules/swarmers/swarmer.dm]:
```suggestion
	fabricated_object.balloon_alert(src, "successfully fabricated")
```

File: code/modules/swarmers/swarmer.dm [hunk 2]
```diff
@@ -374,7 +376,7 @@
 	if(!do_mob(src, src, 10 SECONDS))
 		return
 	adjustHealth(-maxHealth)
-	to_chat(src, span_info("We successfully repaired ourselves."))
+	balloon_alert(src, "sucessfully repaired")
```
Comment (Bibby0110) [inline code/modules/swarmers/swarmer.dm]:
```suggestion
	balloon_alert(src, "successfully repaired")
```

File: code/modules/swarmers/swarmer.dm [hunk 3]
```diff
@@ -389,13 +391,15 @@
 		for(var/d in dronelist)
 			var/mob/living/simple_animal/hostile/swarmer/melee/drone = d
 			drone.set_light(3)
+		balloon_alert(src, "toggled light")
```
Comment (Bibby0110) [inline code/modules/swarmers/swarmer.dm]:
```suggestion
		balloon_alert(src, "light toggled")
```

File: code/modules/swarmers/swarmer.dm [hunk 4]
```diff
@@ -389,13 +391,15 @@
...
 	else
 		set_light(0)
 		if(!mind)
 			return
 		for(var/d in dronelist)
 			var/mob/living/simple_animal/hostile/swarmer/melee/drone = d
 			drone.set_light(0)
+	balloon_alert(src, "toggled light")
```
Comment (Bibby0110) [inline code/modules/swarmers/swarmer.dm]:
```suggestion
	balloon_alert(src, "light toggled")
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**